src/Psy/Formatter/CodeFormatter.php

```diff
@@ -38,8 +42,7 @@ public static function format(\Reflector $reflector)
             $start = $reflector->getStartLine();
             $end   = $reflector->getEndLine() - $start;
 
-            $colors = new ConsoleColor();
-            $colors->addTheme('line_number', array('blue'));
+            $colors = (new ConsoleColorFactory($colorMode))->getConsoleColor();
```

Comment:
Ah, I didn't know that! These [5.3 test failures](https://travis-ci.org/bobthecow/psysh/jobs/103157241) confirm it.